console/console-init/ui/src/modules/device/dialogs/CreateDevice/MetaData.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import {
+  Button,
+  SelectOptionObject,
+  GridItem,
+  Grid
+} from "@patternfly/react-core";
+import { PlusCircleIcon } from "@patternfly/react-icons";
+import { MetaDataHeader } from "./MetaDataHeader";
+import { MetaDataRow } from "./MetaDataRow";
+
+export interface IMetaData {
+  onPropertySelect: (e: any, selection: SelectOptionObject) => void;
+  onChangePropertyInput?: (value: string) => Promise<any>;
+  onPropertyClear: () => void;
+  propertySelected?: string;
+  propertyInput?: string;
+  setPropertyInput?: (value: string) => void;
+}
+
+export const MetaData: React.FC<IMetaData> = ({
+  onPropertySelect,
+  onChangePropertyInput,
+  onPropertyClear,
+  propertySelected,
+  propertyInput,
+  setPropertyInput
+}) => {
+  const [formData, setFormData] = useState([{ defaults: {}, ext: {} }]);
+
+  const handleAddParentClick = () => {
+    setFormData([...formData, { defaults: {}, ext: {} }]);
+  };
+
+  return (
+    <>
+      <MetaDataHeader sectionName="Default properties" />
+      {formData.map((data, index) => {
+        return (
```

Comment:
i think we don't need extra return here, arrow function return itself.

Reply:
Since `{` is used here so return was needed, I have replaced `{` with `(` and skipped the return.